### DISCUSSION TITLE
Where are the apps stored in the volumes?

### DISCUSSION THREAD

**Discussion by HideoKuze** [Budibase/budibase#9717] [Help]
Budibase went down for some reason, so I reran docker-compose, I also changed the ports for bbproxy. When the site came back up my apps were gone. This has never happened before when re building with docker-compose. Are the apps not stored locally in the volumes?

**Comment by melohagan** [comment]
Hey @HideoKuze 

Is this the production docker-compose file that you are using? 

There was a recent change to the development docker compose file that may have caused app loss, but this shouldn't have affected production in anyway.
